app/code/Magento/Catalog/Test/Unit/Ui/Component/Listing/Columns/StockStatusTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
Please add phpdocs for constunts and methods in both test classes. Tests are covered by static tests as well as other module files.

Reply:
@sivaschenko
Description were added.